app/main.ipynb

```diff
@@ -216,15 +282,28 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
-   "source": [
-    "# write your code here"
-   ],
+   "source": "np.std(A, axis=1)",
```

Comment:
The standard deviation calculation by row is correct. Ensure that the axis parameter is set correctly for the intended operation.